dolphinscheduler-task-plugin/dolphinscheduler-task-seatunnel/src/main/java/org/apache/dolphinscheduler/plugin/task/seatunnel/SeatunnelTask.java

```diff
@@ -115,7 +115,8 @@ private String buildCommand() throws Exception {
         args.addAll(buildOptions());
 
         String command = String.join(" ", args);
-        logger.info("SeaTunnel Flink task command: {}", command);
+        String engineType = seatunnelParameters.getEngine().name().toLowerCase();
+        logger.info("SeaTunnel {} task command: {}", engineType, command);
```

Comment:
I think this is unnessnary since the execution engine can be determined from the task definition or the log of seatunnel.

Reply:
logger.info("SeaTunnel Flink task command: {}", command);  // it may be incorrect

Reply:
> logger.info("SeaTunnel Flink task command: {}", command); // it may be incorrect

Why is it incorrect?

Reply:
it may be SeaTunnal Spark task command